shishito/library/modules/runtime/environment/browserstack_control_environment.py

```diff
@@ -1,102 +1,111 @@
+
+        # Add browserstack credentials
+        bs_auth = self.shishito_support.gid('browserstack')
+
+        # prepare pytest arguments into execution list
+        return {
+            '--junitxml=': '--junitxml=' + junit_xml_path,
+            '--junit-prefix=': '--junit-prefix=' + test_result_prefix,
+            '--html=': '--html=' + html_path,
+            '--html-prefix=': '--html-prefix=' + test_result_prefix,
+            '--xbrowser=': '--xbrowser=' + browser,
+            '--xbrowserversion=': '--xbrowserversion=' + browser_version,
+            '--xos=': '--xos=' + os_type,
+            '--xosversion=': '--xosversion=' + os_version,
+            '--xresolution=': '--xresolution=' + resolution,
+            '--browserstack=': '--browserstack=' + bs_auth
+        }
+
     def get_capabilities(self, combination):
-        """Returns dictionary of capabilities for specific Browserstack browser/os combination """
+        """ Returns dictionary of capabilities for specific Browserstack browser/os combination """
+
         build_name = self.shishito_support.gid('build_name')
-        # test_mobile = self.shishito_support.gid('test_mobile')
+
         capabilities = {
             'acceptSslCerts': self.shishito_support.gid('accept_ssl_cert').lower() == 'false',
             'browserstack.debug': self.shishito_support.gid('browserstack_debug').lower(),
             'project': self.shishito_support.gid('project_name'),
             'build': build_name,
-            'name': self.get_test_name() + time.strftime('_%Y-%m-%d')
-        }
-        # TODO take this information (test mobile) from "platform" module
-        # if test_mobile == 'yes':
-        # capabilities.update({
-        #         'device': self.config['device'],
-        #         'platform': self.config['platform'],
-        #         'deviceOrientation': self.config['deviceOrientation'],
-        #         'browserName': self.config['browserName'],
-        #     })
-        # else:
-        capabilities.update({
+            'name': self.get_test_name() + time.strftime('_%Y-%m-%d'),
             'os': self.config.get(combination, 'os'),
             'os_version': self.config.get(combination, 'os_version'),
             'browser': self.config.get(combination, 'browser'),
             'browser_version': self.config.get(combination, 'browser_version'),
-            'resolution': self.config.get(combination, 'resolution'),
-        })
-        return capabilities
-
-    def get_browser_profile(self, browser_type, capabilities):
-        """ Returns updated browser profile ready to be passed to driver """
-        profile = None
-
-        # add extensions to remote driver
-        if self.shishito_support.gid('with_extension'):
-            # browser_profile = self.add_extension_to_browser(browser_type, browser_profile)
-            pass
+            'resolution': self.config.get(combination, 'resolution')
```

Comment:
why you strip end comma?
the reason why this is allowed is better diff when you add another line later